f8a_worker/data_normalizer.py

```diff
@@ -61,7 +61,7 @@ def transform_keys(data, keymap):
                 in_key = in_k
             key = in_key if len(pair) == 1 else pair[1]
             out[key] = value
-
+            
```

Comment:
You are accidentally adding some white spaces here and on other places. Even they don't do any harm, it's a useless change.